signac/contrib/project.py

```diff
@@ -156,6 +156,15 @@ def __init__(self, config=None, _ignore_schema_version=False):
         self._fn_doc = os.path.join(self._rd, self.FN_DOCUMENT)
         self._document = None
 
+        # Prepare Workspace Directory
+        if not os.path.isdir(self._wd):
+            try:
+                _mkdir_p(self._wd)
+            except OSError:
+                logger.error(
+                    "Error occured while trying to create "
```

Comment:
Spelling fix: occured ➡️ occurred
```suggestion
                    "Error occurred while trying to create "
```